Form/ChoiceList/DoctrineChoiceLoader.php

```diff
@@ -0,0 +1,194 @@
+<?php
+/**
+ * (c) Steffen Brem <steffenbrem@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Doctrine\Bundle\PHPCRBundle\Form\ChoiceList;
+
+use Doctrine\Common\Persistence\ObjectManager;
+use Doctrine\ODM\PHPCR\Mapping\ClassMetadata;
+use PHPCR\Util\UUIDHelper;
+use Symfony\Bridge\Doctrine\Form\ChoiceList\EntityLoaderInterface;
+use Symfony\Bridge\Doctrine\Form\ChoiceList\IdReader;
+use Symfony\Component\Form\ChoiceList\ChoiceListInterface;
+use Symfony\Component\Form\ChoiceList\Factory\ChoiceListFactoryInterface;
+use Symfony\Component\Form\ChoiceList\Loader\ChoiceLoaderInterface;
+use Symfony\Component\PropertyAccess\PropertyAccess;
+
+/**
+ * DoctrineChoiceLoader
+ *
+ * @author Steffen Brem <steffenbrem@gmail.com>
+ */
+class DoctrineChoiceLoader implements ChoiceLoaderInterface
```

Comment:
do we really need to duplicate the whole class? could we extend it and just change what is actually different?